src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/custom.py

```diff
@@ -1475,9 +1475,11 @@ def list_vmss_instance_connection_info(resource_group_name, vm_scale_set_name):
     public_ip_address = public_ip.ip_address
 
     # loop around inboundnatrule
-    instance_addresses = []
+    instance_addresses = {}
```

Comment:
Agreed, but could you point out an example using this? Afaik, at core level, we convert to dict a few times, like [here](https://github.com/Azure/azure-cli/blob/master/src/azure-cli-core/azure/cli/core/util.py#L153) and [there](https://github.com/Azure/azure-cli/blob/master/src/azure-cli-core/azure/cli/core/_output.py#L41), so returning an OrderedDict at command level won't make difference